lib/iris/tests/unit/fileformats/pyke_rules/compiled_krb/fc_rules_cf_fc/test_reorder_bounds_data.py

```diff
@@ -56,19 +55,6 @@ def test_slowest_varying(self):
         expected = np.rollaxis(bounds_data, 0, bounds_data.ndim)
         self.assertArrayEqual(res, expected)
 
-    def test_slowest_varying_lazy(self):
```

Comment:
Why did you choose to just remove this test rather than skip it like the others?